pyaerocom/config.py

```diff
@@ -88,6 +88,9 @@ class Config:
     #: DMS
     DMS_AMS_CVO_NAME = "DMS_AMS_CVO"
 
+    #: MEP name
+    MEP_NAME = "MEP"
+
```

Comment:
Why is this needed?
You are coupling back the plugin with something configured on the core.
This breaks the core/plugin separation,

Reply:
If I don't include it, the code just hangs... maybe we can discuss Monday